.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: module",
+            "type": "python",
+            "request": "launch",
+            "module": "behave",
+            "justMyCode": true,
+            "args": ["-D","input='/home/geert/Documents/validation_service/buildingSMART/gherkin/alb003/ifc-gherkin-rules/test/files/alb003/fail-alb003-wrong_direction.ifc'"]
```

Comment:
I'd prefer not to have absolute paths in public projects.

Reply:
I used this launch.json file for debugging in VScode on my local machine. In other branches, I made sure for git to ignore it by putting it into .gitignore. It's deleted and .gitignore modified now